roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -179,6 +179,11 @@ masterPublicURL: {{ openshift.master.public_api_url }}
 networkConfig:
   clusterNetworkCIDR: {{ openshift.master.sdn_cluster_network_cidr }}
   hostSubnetLength: {{ openshift.master.sdn_host_subnet_length }}
+{% if openshift.common.version_gte_3_7 | bool %}
+  clusterNetworks:
+  - cidr: {{ openshift.master.sdn_cluster_network_cidr }}
+    hostSubnetLength: {{ openshift.master.sdn_host_subnet_length }}
```

Comment:
Same here.  osm_host_subnet_length is required now, let's use that directly and cut out openshift_facts.